WcaOnRails/app/models/post.rb

```diff
@@ -2,11 +2,37 @@
 
 class Post < ApplicationRecord
   belongs_to :author, class_name: "User"
+  has_many :post_tags, autosave: true, dependent: :destroy
 
   validates :title, presence: true, uniqueness: true
   validates :body, presence: true
   validates :slug, presence: true, uniqueness: true
 
+  def reload
```

Comment:
I understand this is done to simplify tests, but I'd much rather make the test a little more awkward, like `post.reload` -> `post.find(post.id)`.

That also removes the errorpronability of changing @-cached variables.